model/compile/QtiAssetCompiler/QtiItemAssetCompiler.php

```diff
@@ -101,20 +104,7 @@ private function resolve(ItemMediaResolver $resolver, string $assetUrl, string $
 
     private function getReplacementName(string $link, array $replacementList): string
```

Comment:
```suggestion
    private function getReplacementName(string $link): string
```
Parameter can be removed now

Reply:
also if you do this change please remove line : 53, 64 and 67